src/panels.cpp

```diff
@@ -1695,7 +1695,7 @@ static void draw_health_classic( avatar &u, const catacurses::window &w )
     if( veh ) {
         veh->print_fuel_indicators( w, point( 39, 2 ) );
         mvwprintz( w, point( 35, 4 ), c_light_gray, veh->face.to_string_azimuth_from_north() );
-        veh->print_speed_gauge( w, point( 21, 5 ), 0 );
+        veh->print_speed_gauge( w, point( 21, 5 ), 1 );
```

Comment:
```suggestion
      double speed = convert_velocity( veh->velocity, VU_VEHICLE );
      veh->print_speed_gauge( w, point( 21, 5 ), (-99 < speed < -9 &&99 < speed < 1000) ? 1 : 0 );
```
It's fine while -10 < speed < 100, it's touching stamina label at -99 < speed < -9 and 99 < speed < 1000.